run.py

```diff
@@ -106,6 +126,19 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        try:
+            command = ['ls', '-l', 'issues.db']
```

Comment:
Can we use https://docs.python.org/2/library/os.path.html#os.path.exists instead? That way it will work cross-platform.